src/main/java/exception/EntityNotFoundException.java

```diff
@@ -1,7 +1,13 @@
 package exception;
 
+import java.util.function.Supplier;
+
 public class EntityNotFoundException extends RuntimeException {
     public EntityNotFoundException(String message) {
         super(message);
     }
+
+    public static Supplier<EntityNotFoundException> supplier(String massage) {
+        return () -> new EntityNotFoundException(massage);
+    }
```

Comment:
why do you need that?
```suggestion
    }
```